acquia/acquia_cms
# ACMS-1500: Added module enforced dependency in site studio packages.

**Motivation**
Fixes ACMS-1500

**Proposed changes**
Added module enforced dependency in site studio packages.

**Alternatives considered**
NA

**Testing steps**
- Install site with this code
- Install site studio module import packages
- Now try to uninstall any module having site studio package.
- It should uninstall and delete related packages.
- Verify site.

**Merge requirements**
